### PR TITLE
GEN-1176: Update UI for moving flow

### PR DESCRIPTION
**Pull request by sladan-hedvig**
## Checklist

- [ ] Dark/light mode verification
- [ ] Landscape verification
- [ ] iPad verification
- [ ] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

File: Projects/MoveFlow/Sources/View/MovingFlowHousingType.swift [hunk 1]
```diff
@@ -3,54 +3,74 @@ import SwiftUI
...
 
     public var body: some View {
-
         LoadingViewWithContent(MoveFlowStore.self, [.fetchMoveIntent], [.getMoveIntent]) {
-            hForm {
-                VStack {
-                    ForEach(housingTypes, id: \.self) { type in
-                        setCheckBoxComponent(text: type)
+            hForm {}
+                .hFormTitle(.standard, .title3, L10n.changeAddressSelectHousingTypeTitle)
```
Comment (julia-andersson) [inline Projects/MoveFlow/Sources/View/MovingFlowHousingType.swift:12]: The text font doesn't look the same as in the figma design. Should be standard like it is on the other places

File: Projects/MoveFlow/Sources/View/MovingFlowApartmentView.swift [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+        hFloatingTextField(
+            masking: Masking(type: .address),
+            value: $vm.address,
+            equals: $vm.type,
+            focusValue: .address,
+            error: $vm.addressError
+        )
+    }
+
+    func postalAndSquareField() -> some View {
```
Comment (julia-andersson) [inline Projects/MoveFlow/Sources/View/MovingFlowApartmentView.swift:49]: in the design we have "m2" (square meters) in the right placeholder which is always there. Can we add this?

File: Projects/MoveFlow/Sources/View/MovingFlowApartmentView.swift [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+    @StateObject var vm = MovingFlowViewModel()
+
+    var body: some View {
+        hForm {}
+            .hFormTitle(.standard, .title1, L10n.changeAddressEnterNewAddressTitle)
+            .sectionContainerStyle(.transparent)
+            .hFormAttachToBottom {
+                hSection {
+                    VStack(spacing: 16) {
+                        VStack(spacing: 4) {
```
Comment (julia-andersson) [inline Projects/MoveFlow/Sources/View/MovingFlowApartmentView.swift:17]: When clicking on a field I have no way to dismiss the keyboard which covers the "send in" button :/

File: Projects/MoveFlow/Sources/View/MovingFlowProcessingView.swift [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    }
+
+    private var successView: some View {
+        ZStack(alignment: .bottom) {
+            BackgroundView().ignoresSafeArea()
+            VStack {
+                Spacer()
+                Spacer()
+                VStack(spacing: 16) {
+                    Image(uiImage: hCoreUIAssets.tick.image)
```
Comment (julia-andersson) [inline Projects/MoveFlow/Sources/View/MovingFlowProcessingView.swift:39]: Looks a bit too small. Make bigger according to design :)

**Review by julia-andersson — CHANGES_REQUESTED** [review]

File: Projects/MoveFlow/Sources/View/MovingFlowApartmentView.swift (hunk 2 above)
Comment (sladan-hedvig) [reply Projects/MoveFlow/Sources/View/MovingFlowApartmentView.swift:51]: Should be fine now 👍

**Pull request merged**